runtime/memprof.c

```diff
@@ -47,8 +47,13 @@ static double one_log1m_lambda;
 int caml_memprof_suspended = 0;
 static intnat callstack_size;
 
-static value callback_alloc_minor, callback_alloc_major,
-  callback_promote, callback_dealloc_minor, callback_dealloc_major;
+#define Alloc_minor(tracker) (Field(tracker, 0))
+#define Alloc_major(tracker) (Field(tracker, 1))
+#define Promote(tracker) (Field(tracker, 2))
+#define Dealloc_minor(tracker) (Field(tracker, 3))
+#define Dealloc_major(tracker) (Field(tracker, 4))
+
+static value tracker;
```

Comment:
It'd be handy to have a reminder comment of the OCaml type this relates to (i.e. Gc.Memprof.tracker)

Reply:
Excellent point, I added a comment.